skimage/filters/tests/test_gabor.py

```diff
@@ -77,6 +77,18 @@ def match_score(image, frequency):
     assert responses[1, 1] > responses[1, 0]
 
 
+def test_morlet_zero_sum():
```

Comment:
How long does it take to compute this test function? I am asking, since it shouldn't take too long due to continuous integration.